### PR TITLE
mvp + plus stretch

### PR DESCRIPTION
**Pull request by TamaHills**

### CODE REVIEW

File: Article/Article.js [hunk 1]
```diff
@@ -85,11 +85,92 @@ const data = [
...
+  })
+
+  return article;
+
+}
+window.addEventListener('load', event => {
+  const articles = document.getElementsByClassName("articles")[0];
+  data.forEach( article => articles.appendChild(createArticle(article)));
+});
+
```
Comment (rleslie1015) [inline Article/Article.js]: Awesome job figuring out a solution to the `null` thing!

File: Article/Article.js [hunk 2]
```diff
@@ -85,11 +85,92 @@ const data = [
...
+  const dateLine = document.createElement('p');
+  const button = document.createElement('span');
+  const close = document.createElement('div');
+
+  //Structure Nodes
+  article.appendChild(heading);
+  article.appendChild(close);
+  article.appendChild(dateLine);
+  article.appendChild(button);
+
```
Comment (rleslie1015) [inline Article/Article.js]: great job creating and structuring the elements!

File: Article/Article.js [hunk 3]
```diff
@@ -85,11 +85,92 @@ const data = [
...
+    article.appendChild(paragraph)
+  })
+
+  //Event Listeners
+
+  button.addEventListener('click', event => {
+    let { target: { parentNode } } = event;
+    parentNode.classList.toggle('article-open');
+  })
+
```
Comment (rleslie1015) [inline Article/Article.js]: For a second I was like what is going on here? This is a destructure-ing thing again, right?

Comment (TamaHills) [reply Article/Article.js]: yeah it looks better to me than
```event.target.parentNode.classList.toggle```

**Pull request merged**